packages/autorest.python/autorest/codegen/templates/model_base.py.jinja2

```diff
@@ -726,7 +726,11 @@ def _get_deserialize_callable_from_annotation(  # pylint: disable=R0911, R0915,
     ):
         if obj is None:
             return obj
-        return _deserialize_with_callable(deserializer, obj)
+        try:
+            return _deserialize_with_callable(deserializer, obj)
+        except Exception:
+            pass
+        return obj
```

Comment:
When use `_deserialize_default`, it means SDK have to use default type to deserialize. For some type (e.g. JSON), it is only for typing so we just need to return raw value.